.github/workflows/continuous-integration-e2e.yaml

```diff
@@ -65,8 +65,8 @@ jobs:
         CHAIN_HISTORY_PROVIDER: 'http'
         CHAIN_HISTORY_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/chain-history"}'
         DB_SYNC_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/cexplorer'
-        PROJECTION_PG_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/projection'
-        PROJECTION_TEST_PG_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/projection_test'
+        STAKE_POOL_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/stake_pool'
```

Comment:
I think `STAKE_POOL_CONNECTION_STRING` (sounds like the conn string of the pool) -> `STAKE_POOL_DB_CONNECTION_STRING` would be one way better, however, it's not a requested change, just a preference